marimo/_islands/island_generator.py

```diff
@@ -397,12 +398,12 @@ def render_init_island(self) -> str:
             <marimo-island
                 data-app-id="{self._app_id}"
                 data-cell-id="{init_cell_id}"
-                data-reactive="{json.dumps(True)}"
+                data-reactive="{json.dumps(False)}"
             >
                 <marimo-cell-output>
                 {init_output}
                 </marimo-cell-output>
-                {init_input}
```

Comment:
Was this just a NameError before? Maybe a unit test to catch this branch

Reply:
But also, is init_island even needed with the toast?

Reply:
> Was this just a NameError before? Maybe a unit test to catch this branch

It wasn't a name error. there was no error, I just inlined the variable. ill create a snapshot test

> is init_island even needed with the toast

they somewhat have different use-cases. `init_island` shows a spinner while HTML loads (seems like you may want to show before any static HTML appears). the toast happens only when you interact with static HTML before it "rehydrates"